docs/docs/reference/05-oauth-providers/mercado-libre.md

```diff
@@ -0,0 +1,35 @@
+---
+id: mercado-libre
+title: Mercado Libre
+---
+
+:::note
+Mercado Libre is currently the only built-in provider using the OAuth 2.0 spec.
+:::
+
+## Documentation
+
+https://developers.mercadolibre.com.ar/es_ar/autenticacion-y-autorizacion
```

Comment:
Should probably link to the English version for bigger reach: https://developers.mercadolibre.com.ar/en_us/authentication-and-authorization